generators/client-2/templates/src/main/webapp/app/account/_account.ng2module.ts

```diff
@@ -17,9 +18,36 @@ import { SessionsComponent } from './sessions/sessions.component';
 <%_ } _%>
 import { SettingsComponent } from './settings/settings.component';
 
+import { settingsState } from './settings/settings.state';
+<%_ if (authenticationType === 'session') { _%>
+import { sessionsState } from './sessions/sessions.state';
+<%_ } _%>
+import { activateState } from "./activate/activate.state";
+import { passwordState } from "./password/password.state";
+import { finishResetState } from "./password-reset/finish/password-reset-finish.state";
+import { requestResetState } from "./password-reset/init/password-reset-init.state";
+import { registerState } from "./register/register.state";
+import { accountState } from "./account.state";
+
+
+
+let ACCOUNT_STATES = [
+    accountState,
+    activateState,
+    passwordState,
+    finishResetState,
+    requestResetState,
+    registerState,
+    <%_ if (authenticationType === 'session') { _%>
+    sessionsState,
+    <%_ } _%>
+    settingsState,
+];
+
 @NgModule({
     imports: [
-        <%=angular2AppName%>SharedModule
+        <%=angular2AppName%>SharedModule,
+        UIRouterModule.forChild({ states: ACCOUNT_STATES })
```

Comment:
By declaring the states on the module, the states will be automatically registered when the module loads.